src/main/java/com/alkemy/ong/application/exception/ExceptionHandler.java

```diff
@@ -1,8 +1,25 @@
 package com.alkemy.ong.application.exception;
 
+import com.alkemy.ong.application.rest.response.ApiErrorResponse;
+import java.util.Arrays;
+import org.springframework.http.HttpHeaders;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
 import org.springframework.web.bind.annotation.ControllerAdvice;
+import org.springframework.web.context.request.WebRequest;
+import org.springframework.web.servlet.mvc.method.annotation.ResponseEntityExceptionHandler;
 
 @ControllerAdvice
-public class ExceptionHandler {
+public class ExceptionHandler extends ResponseEntityExceptionHandler {
 
+  @org.springframework.web.bind.annotation.ExceptionHandler
+  protected ResponseEntity<Object> handleEntityNotFound(RuntimeException ex, WebRequest request) {
```

Comment:
Si estoy haciendo handler de una EntityNotFound, entonces el parámetro del método debe ser EntityNotFound. 

De la forma en que esta escrito este método, todas las exceptions que hereden de RuntimeException devolveran una lista de errores que digan "Entity not found" con un BAD_REQUEST, lo cual no es correcto ni apropiado para e estándar de REST.


CLEAN CODE: Adicionalmente, el parámetro WebRequest no es utilizado en el contexto de este método, por favor remover cosas que no son utilizadas.